brave-core/src/test/java/com/github/kristofa/brave/ITBrave.java

```diff
@@ -107,7 +108,11 @@ public Integer call() throws Exception {
             assertTrue(localSpan.id != 0);
             assertEquals(localSpanName, localSpan.name);
 
-            assertEquals("Span ids should be different.", 3, Stream.of(serverSpan.id, clientSpan.id, localSpan.id).distinct().count());
```

Comment:
Well, I am surprised if it was only one test!

Projects who are multimodule and leave one in an old source version are
kind of gonna have this coming anyway.